tests/unit/test_id.py

```diff
@@ -37,10 +37,14 @@ def test_base(self):
         assert test.id is None
 
     def test_manually_add_id(self):
+        # TODO: it's really not possible to use fixtures with methods??
```

Comment:
should be possible, however this is unittest and not pytest, so that might be the problem? :)

Reply:
It's still executed by pytest... Anyway, it's a problem for another day, I'll be glad to cut out one use of `fmf.log`, and I can address the fixture issue later.